nomenclature/definitions/variable/fuel_types.yaml

```diff
@@ -0,0 +1,124 @@
+# This list of fuels will replace any occurrence of `<Fuel>`
+# in a variable-definition yaml file
+
+# All fuels that have the attribute "ccs: True" have two sub-categories
+# "w/ CCS": with a CO2 capture component
+# "w/o CCS": with freely vented CO2 emissions
+
+<Fuel>:
+  Biomass:
+    description: Purpose-grown biomass, crop residues, forest industry waste,
+      solid waste from domestic, agricultural or municipal sources
+    ccs: True
+
+  Biomass|Traditional:
+    description: Traditional biomass (domestic and agricultural waste,
+      forest residues)
+    ccs: True
+
+  Biomass|New and Imported:
+    description: Biomass from dedicated energy crops and biomass imports
+    ccs: True
+
+  Fossil:
+    description: All fossil fuels (crude oil, coal, natural gas)
+    ccs: True
+
+  Coal:
+    description: Coal (including hard coal and lignite)
+    ccs: True
+
```

Comment:
@danielhuppmann I have a question: how do you declare the variable in the data file when eg it is Coal without CCS?

Reply:
Thank you for this question @sandrinecharousset because it shows that the proposed notation is not yet sufficiently intuitive. The idea is that every fuel in `fuel_types.yaml` that has the attribute `ccs: True` has two sub-categories.

So following your example, there would be three variables:
- `Secondary Energy|Electricity|Coal` (which is the total)
- `Secondary Energy|Electricity|Coal|w/ CCS`
- `Secondary Energy|Electricity|Coal|w/o CCS`

This is (tried to be) explained at the top of that file. Can you propose how to rename the attribute or extend the description to make this immediately obvious?

Reply:
Thanks @danielhuppmann 
When reading again the file, I find it clear. But I propose to add just examples so that too-quick readers (such as myself) will catch it....

Reply:
thank you @sandrinecharousset - I rewrote it to make it even clearer that we are talking about three variables, not just the two sub-categories. hope you agree!